examples/favorites_advanced/bloc_favorites_advanced/lib/feature_puppy/search/views/search_page.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:bloc_sample/feature_puppy/search/blocs/puppy_list_bloc.dart';
+import 'package:favorites_advanced_base/core.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:favorites_advanced_base/resources.dart';
+import 'package:provider/provider.dart';
+
+class SearchPage extends StatelessWidget {
+  static const searchPage = 'Search page';
+
+  @override
+  Widget build(BuildContext context) =>
+      BlocBuilder<PuppyListBloc, PuppyListState>(
+          // buildWhen: (previous, current) {
+          //   final result = previous.status != current.status;
+          //   print('buildWhen current.status : ${current.status}');
+          //   print('RESULT : $result');
+          //   return result;
+          // },
+          key: const Key(Keys.puppySearchPage),
+          builder: (context, state) {
+            print('BlocBuilder builder: ${state.status}');
+            switch (state.status) {
+              case PuppyListStatus.initial:
+                return puppyListStatusInitial(context);
```

Comment:
@StanislavPrime, please use the following conventation
1. All the internal builders should be marked as private -> `_`
2. All the builders should start with`_build{Action}`, for instance `_buildPuppyListStatusInitial`

Reply:
I have changed the internal builders in this file to start with _